### ISSUE TITLE
Unable to easily disable CSRF checks

### ISSUE THREAD

**Issue by MarcinOrlowski** [skipperbent/simple-php-router#85]
There's no easy way to disable CSRF check but to write dummy CsrfVerifer implementation as     neither SimpleRouter's csrfVerifier() nor RouterBase's `setBaseCsrfVerifier()` accept `null`, even if rest of the code checks for `null`.

**Comment by skipperbent** [comment]
Actually I noticed this aswell yesterday - it caused some issue in projects where CSRF was not necessary.

So in the next release `RouterBase` will no longer instantiate a CsrfVerifier per default. If someone want to use the CsrfVerifier feature, they'll need to add it themselve by using `setBaseCsrfVerifier` in their `routes.php` etc.

**Comment by skipperbent** [comment]
This is fixed in version 1.7 - please read the release notes if you like to implement the CSRF security feature:

CSRF-token are no longer initiated by `RouterBase`. That means that the router will have no CSRF security enabled by default. To add this functionality, please add this line to your custom router-class or `routes.php` file:

```php
SimpleRouter::csrfVerifier(new \Pecee\Http\Middleware\BaseCsrfVerifier());
```
